### PR TITLE
[BXMSPROD-1805] Jenkins job for kogito sources generation

### PR DESCRIPTION
**Pull request by lampajr**
**Thank you for submitting this pull request**

**JIRA**: 
- https://issues.redhat.com/browse/BXMSPROD-1805

Created new jenkins job for Kogito container sources generation.

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
+* Create the OpenShift images templates and sources
```
Comment (rgdoliveira) [inline job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy:2]: Remove the templates string from the comment.

File: job-dsls/src/main/resources/job-scripts/prod_kogito_create_images_templates_and_sources.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
+pipeline {
```
Comment (rgdoliveira) [inline job-dsls/src/main/resources/job-scripts/prod_kogito_create_images_templates_and_sources.jenkinsfile:1]: How about renaming the jenkinsfile to **prod_kogito_create_images_sources.jenkinsfile**

File: job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/**
```
Comment (rgdoliveira) [inline job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy:1]: How about renaming the groovy to **prod_kogito_create_images_sources.groovy**

File: job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Create the OpenShift images templates and sources
+*/
+def scriptTemplate = this.getClass().getResource("job-scripts/prod_kogito_create_images_templates_and_sources.jenkinsfile").text
+def parsedScript = scriptTemplate.replaceAll(/<%=\s*(\w+)\s*%>/) { config[it[1]] ?: '' }
+
+def folderPath = "PROD"
+folder(folderPath)
+
+pipelineJob("${folderPath}/kogito-create-container-sources") {
```
Comment (rgdoliveira) [inline job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy:10]: the rhba job is called **create-images-templates-and-sources**, what if we call this one **kogito-create-images-sources** to keep somehow aligned.

**Review by rgdoliveira — APPROVED** [review]

**Comment by lampajr** [comment]
Jenkins retest this

**Pull request merged**